Library/Client.ts

```diff
@@ -131,17 +132,22 @@ class Client {
         this.track(data);
     }
 
-    public trackRequestSync(request: http.ServerRequest, response: http.ServerResponse, ellapsedMilliseconds?: number, properties?: { [key: string]: string; }, error?: any) {
-        RequestTracking.trackRequestSync(this, request, response, ellapsedMilliseconds, properties, error);
+    public trackServerRequestSync(request: http.ServerRequest, response: http.ServerResponse, ellapsedMilliseconds?: number, properties?: { [key: string]: string; }, error?: any) {
+        ServerRequestTracking.trackRequestSync(this, request, response, ellapsedMilliseconds, properties, error);
     }
 
-    public trackRequest(request: http.ServerRequest, response: http.ServerResponse, properties?: { [key: string]: string; }) {
-        RequestTracking.trackRequest(this, request, response, properties);
+    public trackServerRequest(request: http.ServerRequest, response: http.ServerResponse, properties?: { [key: string]: string; }) {
```

Comment:
Yes, but what about the `trackClientRequest` method I have added here? Without that, it would be difficult for an application to *manually* track HTTP dependency requests, because it takes a few dozen non-obvious lines of code to convert a `http.ClientRequest` into appropriate parameters for `trackDependency`. (Perhaps it could be renamed `trackDependencyRequest` instead of `trackClientRequest`.)